### ISSUE TITLE
Use of your library

### ISSUE THREAD

**Issue by ghost** [alextaujenis/RBD_Timer#19]
Hi alextaujenis,

I would like to use this library in some projects I am working on and noticed that this library is under a MIT License.

Would you kindly advise what requirements I need to do to ensure compliance with this license.

Regards and have a great day
George

**Comment by alextaujenis** [comment]
Hi @gxdeange,

You can cite the Copyright and MIT license many different ways.

* Include the library copyright header in your source code
* Include the copyright notice in your README.md
* Include the copyright notice in some other file in your repo
* Include a full copy of the RBD library - including copyright and license - bundled with your application
* Add a section in your public application that users can access with the full copyright and MIT license
* Add a section to your public website with a reference to the copyright notice and MIT license
* Add the copyright as a footnote in a book or paper you are publishing
* Add the copyright and full MIT license as a reference in a book or paper you are publishing
* Leave it out entirely (not recommended)

I'm not a license expert, and this is not a complete (or possibly even correct) list of how to cite libraries under the MIT license.

Basically the game here is I don't want you to sue me for any reason when I give you my software for free, and you need to inform others that when they use my software, via proxy of your software, that they have given up all rights to sue me as well. It's a legal disclaimer. It's also nice to include the reference so anyone looking over your software in the future can:

a) Look up the source code
b) Find the docs online
c) Bump outdated versions in your code
d) Bother me when something breaks

If you'd like to give more information about your situation; I could provide a more specific answer (which may or may not be technically correct). If this rant answers your question then please close this issue.

Thank you for using this library and actually taking the time to consider the license!

**Comment by ghost** [comment]
Thanks @alextaujenis,

I am using this library for some custom designed and made stepper and brushed motor programmable boards.

The idea (and now reality) is that when the motor stops rotating or has been disabled is that the cooling fan will continue to run for a few more seconds.

I did notice that sometimes the run on was not the set timeout but worked it out by adding a few timer.restart() inside my very simple control code.

Cheers and have a great day
